models/state.go

```diff
@@ -54,7 +54,7 @@ func (s State) TransitionAllowed(target State) bool {
 		}
 	case StateImporting:
 		switch target {
-		case StateImported, StateFailedImport, StateDeleted:
+		case StateImporting, StateImported, StateFailedImport, StateDeleted:
```

Comment:
Is this right? I believe a transition where the state hasn't changed is not a transition. It suggests that the same action has been performed multiple times.

Reply:
I added it here because the import refers to a download variant, but another option would be to check the state in the main function, and if it is already `importing` we don't update it.